locales/en.json

```diff
@@ -2023,7 +2035,14 @@
 			"advertisementForOurTeam": "Do you associate Node, React and Feathers with Javascript rather than English? Are you looking for a young team, a relaxed atmosphere and flat hierarchies? Then take a look at our vacancies: https:\/\/schul-cloud.org\/community#jobs",
 			"browserDoesNotSupportPlayer": "Your browser does not support this (HTML5) video player.",
 			"downloadFile": "Download file",
+			"dataCompliantAndSecure": "The {{title}} is developed in and for Germany. Data protection-compliant access to learning software, as well as data sovereignty, are ensured.",
+			"developedWithAndForSchools": "The {{title}} is optimized for what teachers and students need and can be connected with learning offers from many providers. Together with our users and numerous partners in the German education landscape, we continue to develop!",
+			"easyToUse": "The {{title}} is secure, transparent and can be used without a long training period - independent of time and with your own cell phone, tablet or computer.",
+			"featuresAndMore": "Learning materials, chat, assignments, video conferencing and more",
+			"freeForAllSchools": "Free for all schools",
+			"internationalShapingDigitalEducationInPartnership": "Together with our strong partners, the German schools abroad, supported by the German Foreign Office, the {{title}} is being further developed.",
```

Comment:
"The SC is developed in partnership with the World Association of German Schools Abroad and officially supported by the German Foreign office"